lib/zlib.js

```diff
@@ -54,8 +53,8 @@ const codes = {
 };
 
 const ckeys = Object.keys(codes);
-for (var ck = 0; ck < ckeys.length; ck++) {
-  var ckey = ckeys[ck];
+
+for (const ckey of ckeys) {
```

Comment:
So, can you do the benchmark or do I have to do it (by the way I have no idea of how to run them)